### PR TITLE
Adds LICENSE.md

### PR DESCRIPTION
**Pull request by tallysmartins**
I'm adding this based on the license of [Jason project](https://github.com/michalmuskala/jason)

### CODE REVIEW

**Comment by tallysmartins** [comment]
If this is the correct license I can add to the other repos too if you want.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2017 Michał Muskała
```
Comment (michalmuskala) [inline LICENSE.md]: Let's rather use "The ElixirBench team" - it's not just me 🙂

Comment (tallysmartins) [reply LICENSE.md]: My mistake! Done! :+1:

**Comment by tallysmartins** [comment]
I'm wondering now if there's a way to add the license file up to the organization level....

**Comment by michalmuskala** [comment]
Thank you! :heart:

I think having it in each project is fine - there's not as many of them.

**Pull request merged**